ui/package.json

```diff
@@ -32,16 +33,16 @@
     "start": "npm-run-all -p watch-css start-js",
     "build-js": "react-scripts build",
     "build": "npm-run-all build-css build-js",
-    "test": "react-scripts test --env=jsdom",
-    "coverage": "react-scripts test --env=jsdom --coverage",
+    "test": "react-scripts test --env=jsdom --transformIgnorePatterns \"node_modules/(?!lodash-es)/\"",
+    "coverage": "react-scripts test --env=jsdom --coverage --transformIgnorePatterns \"node_modules/(?!lodash-es)/\"",
```

Comment:
@jhellar I think you were having some problems with using `lodash` in the tests before, right? Does this solve your problem?

Reply:
@wei-lee yes, I think so. there was just one place where I used lodash, so I replaced it with plain  js, but good to have this

Reply:
Is it bad to use `lodash`? I really like to use the get, set and other useful methods, that are pretty messy in plain Javascript.

Reply:
@wojta no, no problem at all to use in the test. It's just @jhellar was trying to do the same in the tests before and there was some issues with it. This helps solve his problem so I just ping him here.